[2022-01-18 10:32:34,712] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: example_dag.dbt_trigger 2022-01-17T00:00:00+00:00 [queued]>
[2022-01-18 10:32:34,718] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: example_dag.dbt_trigger 2022-01-17T00:00:00+00:00 [queued]>
[2022-01-18 10:32:34,718] {taskinstance.py:1094} INFO - 
--------------------------------------------------------------------------------
[2022-01-18 10:32:34,718] {taskinstance.py:1095} INFO - Starting attempt 1 of 1
[2022-01-18 10:32:34,718] {taskinstance.py:1096} INFO - 
--------------------------------------------------------------------------------
[2022-01-18 10:32:34,729] {taskinstance.py:1114} INFO - Executing <Task(BashOperator): dbt_trigger> on 2022-01-17T00:00:00+00:00
[2022-01-18 10:32:34,732] {standard_task_runner.py:52} INFO - Started process 181 to run task
[2022-01-18 10:32:34,736] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'example_dag', 'dbt_trigger', '2022-01-17T00:00:00+00:00', '--job-id', '118', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/example_dag.py', '--cfg-path', '/tmp/tmpwmxh8y60', '--error-file', '/tmp/tmpy27gw6mj']
[2022-01-18 10:32:34,737] {standard_task_runner.py:77} INFO - Job 118: Subtask dbt_trigger
[2022-01-18 10:32:34,779] {logging_mixin.py:109} INFO - Running <TaskInstance: example_dag.dbt_trigger 2022-01-17T00:00:00+00:00 [running]> on host 756c785ba7b5
[2022-01-18 10:32:34,857] {taskinstance.py:1251} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=example_dag
AIRFLOW_CTX_TASK_ID=dbt_trigger
AIRFLOW_CTX_EXECUTION_DATE=2022-01-17T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-17T00:00:00+00:00
[2022-01-18 10:32:34,858] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2022-01-18 10:32:34,858] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'pwd']
[2022-01-18 10:32:34,865] {subprocess.py:74} INFO - Output:
[2022-01-18 10:32:34,867] {subprocess.py:78} INFO - /tmp/***tmps29ffzj2
[2022-01-18 10:32:34,867] {subprocess.py:82} INFO - Command exited with return code 0
[2022-01-18 10:32:34,894] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=example_dag, task_id=dbt_trigger, execution_date=20220117T000000, start_date=20220118T103234, end_date=20220118T103234
[2022-01-18 10:32:34,914] {local_task_job.py:151} INFO - Task exited with return code 0
[2022-01-18 10:32:34,933] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
